MvvmCross/Platforms/Uap/Presenters/MvxWindowsViewPresenter.cs

```diff
@@ -23,6 +23,8 @@ public class MvxWindowsViewPresenter
     {
         protected readonly IMvxWindowsFrame _rootFrame;
 
+        private MvxWindowsContentDialog _windowsContentDialog;
```

Comment:
I don't think it is good to keep an instance reference like this. We probably need something to find the view back in the stack.

Reply:
Made.